.github/workflows/release.yml

```diff
@@ -1,18 +1,31 @@
-name: Automated Release
+name: Release
 
 on:
   schedule:
     # Runs at 2:00 AM UTC on the first Saturday of every month
-    - cron: '0 2 * * 6'
-  workflow_dispatch: # Allow manual triggering of the workflow
+    - cron: '0 2 1-7 * 6'
```

Comment:
See https://crontab.guru/#0_2_1-7_*_6

Reply:
Splitting this part out into a smaller PR in case the rest of this PR takes longer to merge: https://github.com/jesseduffield/lazygit/pull/4308